include/bluetooth/gatt.h

```diff
@@ -1669,6 +1669,13 @@ struct bt_gatt_subscribe_params {
 #endif /* CONFIG_BT_GATT_AUTO_DISCOVER_CCC */
 	/** Subscribe value */
 	uint16_t value;
+#if defined(CONFIG_BT_SMP)
+	/** Minimum required security for received notification. If not set
+	 * explicitly by application it is set by host based on security level
+	 * at the time of subscription.
+	 */
+	bt_security_t min_security;
```

Comment:
Did you consider the possibility of having the stack automatically populate this value (i.e. remove it from the API), so that it matches the security level of the CCCD of the server instead?

Reply:
not sure were this would to be stored if not in subscribe params (it needs to be provided on resubscribe too)

Reply:
Could it be stored in the `ccc_store` struct?

Reply:
not sure if I follow,  ccc_store is for server, right?